tests/testthat/test-utils.R

```diff
@@ -88,6 +88,31 @@ test_that("parse_database_error() works with messages from the wild", {
           <SQL> 'SELECT DISTINCT \"po_id\", ***CENSORED***'"
   expect_snapshot(error = TRUE, rethrow_database_error(msg, call = NULL))
 
+  # Really long error message can cause rethrow_database_error to take a really long time.
```

Comment:
```suggestion
  skip_on_cran()
  
  # Really long error message can cause rethrow_database_error to take a really long time.
```

I add this test skip on CRAN as I've received some feedback from a CRAN maintainer in the past that a failure w.r.t. an arbitrarily chosen timing violated the policy "The time of the volunteers is CRAN’s most precious resource"🙃